File: app/src/main/java/com/parabola/newtone/mvp/view/ColorThemeSelectorView.java
package com.parabola.newtone.mvp.view;

import com.parabola.domain.settings.ViewSettingsInteractor.ColorTheme;
import com.parabola.domain.settings.ViewSettingsInteractor.PrimaryColor;

import moxy.MvpView;
import moxy.viewstate.strategy.alias.AddToEndSingle;

@AddToEndSingle
public interface ColorThemeSelectorView extends MvpView {

    void setDarkLightTheme(ColorTheme colorTheme);
    void setPrimaryColor(PrimaryColor primaryColor);
}

File: app/src/main/java/com/parabola/newtone/mvp/view/ColorThemeSelectorView.kt
package com.parabola.newtone.mvp.view

import com.parabola.domain.settings.ViewSettingsInteractor.ColorTheme
import com.parabola.domain.settings.ViewSettingsInteractor.PrimaryColor
import moxy.MvpView
import moxy.viewstate.strategy.alias.AddToEndSingle

@AddToEndSingle
interface ColorThemeSelectorView : MvpView {
    fun setDarkLightTheme(colorTheme: ColorTheme)
    fun setPrimaryColor(primaryColor: PrimaryColor)
}
